db/controllers/articles.controller.js

```diff
@@ -38,8 +38,11 @@ exports.updateArticleVote = (req, res, next) => {
 
   patchArticleVote(articles_id, inc_votes)
     .then((article) => {
-      res.status(201).send({article})
+      if (!article) {
+        return Promise.reject({ status: 404, msg: "Page Not Found" });
```

Comment:
Something like "article not found" or "resource not found" could go here instead

Reply:
- [ X ]  as above